src/components/FindPark.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <section>
+    <h2>Let's Go Play!</h2>
+    <button @click="searchByLocation">Find a dog park near me!</button>
+    <h3>--Or--</h3>
+    <input
+      class="search-input"
+      type="text"
+      placeholder="Search by name, city or zip code"
+      ref="enteredValue"
+    />
+    <button class="search-button" @click="setSearchTerm">
+      Get Started - woof!
+    </button>
+    <p v-if="inputIsInvalid">Please enter a park name, city, or zip code</p>
+  </section>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      searchTerm: '',
+      inputIsInvalid: false,
```

Comment:
Nice addition here with `inputIsInvalid`.  Makes for cleaner code within `v-if` by evaluating for a truthy value.